### DISCUSSION TITLE
请问页码在哪儿？怎样才能让页码居中显示呢？

### DISCUSSION THREAD

**Discussion by shuyangwei** [TheNetAdmin/zjuthesis#405] [General]
如题。感谢！

**Comment by Jackjiang313** [comment]
config-->format-->general-->layout.tex
把\TwoSide的条件语句中的\fancyfoot[OR,EL]{\zihao{-5} \songti \thepage}改为\fancyfoot[C]{\zihao{-5} \songti \thepage}

**Reply by shuyangwei** [reply]
成功了！感谢！！
